source/faq/questions/dashboard-faq.md

```diff
@@ -82,6 +82,12 @@ We have already begun the implementation for capturing even more things from you
 
 These will be added in subsequent releases.
 
+## {% fa fa-angle-right %} Why are my video recordings freezing or dropping frames when running in CI?
+
+There is a {% url 'known issue' https://github.com/cypress-io/cypress/issues/4722 %} where videos recorded to the Dashboard may freeze up or drop frames if there are not enough resources available when running the tests in your CI container. You can run your tests with {% url 'memory and CPU logs enabled' troubleshooting#Log-memory-and-CPU-usage %} to identify and evaluate the resource utilization within your CI.
+
+If you are experiencing this issue, we recommend switching to a more powerful CI container or provider. There is also an {% url 'open issue' https://github.com/cypress-io/cypress/issues/5061 %} to add detect and warn when a video recording freezes or drop frames.
```

Comment:
"to add detect and warn when ..." is weird. Maybe "to add dropped frames and video recording freezes detection, and warn the user"? Not sure.

Reply:
I'm not sure I would link to this open issue either. This also still makes it sound like this is a Cypress bug, but this issue would really only help them with debugging their existing CPU problems.